CHANGELOG.md

```diff
@@ -29,6 +29,11 @@ This is an example of a brief overview of the _Major_ or _Minor_ version changes
 
 ---
 
+## [4.31] Added IconButton and StackedButton components
```

Comment:
merged `4.31` earlier in the day - will need to bump the minor again

Reply:
done